content/applications/inventory_and_mrp/manufacturing/workflows/continuous_improvement.rst

```diff
@@ -0,0 +1,233 @@
+   configured to meet the needs of almost any workflow.
+
+   As such, it is important to remember the content below only provides *examples* of how Odoo
+   *might* be used. They should be viewed as more of a starting point, rather than a concrete
+   outline that every organization must follow.
+
+Identify problems
+=================
+
+Before improvement can begin, it is necessary to determine where improvement is necessary. This is
+where identifying problems comes into play. Two of the best Odoo apps for identifying problems with
+products or processes are *Helpdesk* and *Quality*.
+
+Helpdesk
+--------
+
+The *Helpdesk* app is useful for receiving feedback from outside of the organization, like from
+clients or customers. This is accomplished by implementing one (or more) of the methods for
+:doc:`receiving tickets <../../../services/helpdesk/overview/receiving_tickets>`, including email
+aliases, live chat conversations, and website forms.
+
+Using these methods, customers can submit feedback about problems, which is then reviewed by a
+member of a :doc:`helpdesk team <../../../services/helpdesk>`. Depending on the outcome of the
+review, the team member may decide to take further action to ensure the issue is addressed. This can
+include creating a :doc:`quality alert <../../quality/quality_management/quality_alerts>`.
+
+Quality
+-------
+
+The Quality app is useful for receiving feedback from *within* the organization, like from
+employees.
+
+One method for accomplishing this is to set up a :doc:`quality control point
+<../../quality/quality_management/quality_control_points>` (QCP). A |QCP| is used to automatically
+create quality checks at regular intervals, prompting employees to inspect, and confirm, the quality
+of a product.
+
+If an issue is found, an employee can then create a :doc:`quality alert
+<../../quality/quality_management/quality_alerts>` to notify a quality team. Quality alerts can also
+be created independent of a |QCP|, in the event that an employee discovers an issue without being
+prompted to check for one. This is a great way for customer support employees to notify a quality
+team of an issue brought to their attention by a customer ticket.
+
+Suggest improvements
+====================
+
+Once a problem is identified, the next step is to put forward ideas for how to address the problem.
+As with identifying problems, the Quality app is also useful for suggesting improvements. In
+addition, the *PLM* (*Product Lifecycle Management*) app can be used for this purpose, as well.
+
+Quality
+-------
+
+When creating a quality alert to bring an issue to the attention of a quality team, the *Corrective
+Actions* and *Preventive Actions* tabs can be used to provide feedback about how the issue can be
+addressed.
+
+The Corrective Actions tab is used to suggest a method for fixing items affected by the issue. For
+example, `Screw the bolts on tighter so that the seat stays in place`.
```

Comment:
If the ref was used in my suggestion above, these can be :guilabel:, otherwise please use italics when referencing a field that is not currently on the UI, even if it was italicized previously (this differs from introducing a term for the first time, since we are referencing a UI field). 
```suggestion
The :guilabel:`Corrective Actions` tab is used to suggest a method for fixing items affected by the
issue. For example, `Screw the bolts on tighter so that the seat stays in place`.
```